planning/behavior_path_planner/src/utils/utils.cpp

```diff
@@ -1212,6 +1212,17 @@ std::vector<DrivableLanes> generateDrivableLanesWithShoulderLanes(
   return drivable_lanes;
 }
 
+std::vector<DrivableLanes> getNonOverlappingExpandedLanes(
+  PathWithLaneId & path, const std::vector<DrivableLanes> & lanes,
+  const DrivableAreaExpansionParameters & parameters)
+{
+  const auto shorten_lanes = utils::cutOverlappedLanes(path, lanes);
+  const auto expanded_lanes = utils::expandLanelets(
+    shorten_lanes, parameters.drivable_area_left_bound_offset,
+    parameters.drivable_area_right_bound_offset, parameters.drivable_area_types_to_skip);
+  return expanded_lanes;
```

Comment:
```suggestion
  return utils::expandLanelets(
    shorten_lanes, parameters.drivable_area_left_bound_offset,
    parameters.drivable_area_right_bound_offset, parameters.drivable_area_types_to_skip);
```
I would suggest do directly return the expanded lanes.